FAQ.md

```diff
@@ -60,4 +60,39 @@ In this case make sure to Dispose of each instance when finished with it.
 
 ### How do I create a nuget package from a discovery doc?
 
-See [HowToGenerateAPackage.md](HowToGenerateAPackage.md).
\ No newline at end of file
+See [HowToGenerateAPackage.md](HowToGenerateAPackage.md).
+
+### How do I populate a property of type `GoogleApiHttpBody` in a request?
+
+Some libraries (e.g. Cloud Machine Learning Engine) require un-typed
+JSON to be passed through to the server; ignoring the strongly-typed
+generated code. This un-typed JSON is represented via the `GoogleApiHttpBody`
+type, usually for a property called `HttpBody`.
+
+Setting the data in this property directly does not work as
+expected. The workaround is to set the content directly in the
+`HttpRequestMessage` via an interceptor. For example:
+
+```csharp
+DemoService service = new DemoService(...);
+
+// The JSON we want in our HTTP request body
+string json = "{ \"property\": 5 }";
+
+// Prepare to send the request
+string actualBodyJson = "Request JSON";
```

Comment:
`actualBodyJson` doesn't seem to be used?

Reply:
Whoops - that was left over from the issue example. Will remove,